tests/test_warc_download.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2019 CERN.
+#
+# CERN Analysis Preservation is free software; you can redistribute it
```

Comment:
Please replace by `twikiget` here and everywhere in the file headers license statements.  You can also fit it in two lines like for REANA.